geomesa-core/src/main/scala/org/locationtech/geomesa/core/stats/StatWriter.scala

```diff
@@ -60,7 +61,7 @@ object StatWriter extends Runnable with Logging {
 
   private val writeDelayMillis = 1000
 
-  private val batchWriterConfig = new BatchWriterConfig().setMaxMemory(10000L).setMaxWriteThreads(5)
+  private val batchWriterConfig = GeoMesaBatchWriterConfig().setMaxMemory(10000L).setMaxWriteThreads(5)
```

Comment:
ditto max memory... although now that i think about it, maybe we'd want to cap stat writer and metadata writers at lower amounts of memory, and have the config only affect the main geomesa feature writing.